scripts/cspell/project-words.txt

```diff
@@ -14,7 +14,7 @@ certfile
 changeme
 clickhouse
 cncf
-conrolled
+controlled
```

Comment:
Are these words even necessary in the dictionary?

Reply:
Not sure, can you please suggest either those words [diligence,distributed,documenting,controlled] should be removed from this file or leavr untouched?

Reply:
If the linter is successful without having these words in the dictionary we should remove them.

Reply:
I have reorder that list alphabetically due to which it was failing previously.  Can you please check